README.md

```diff
@@ -3,3 +3,5 @@
 A scraper for https://parlament.gv.at
 
 To scrape the data of all MPs of the Austrian Nationalrat run `scrapy runspider spider.py -o mps.json` in the top directory.
+
+You can convert the resulting JSON to a CSV with `python3 json2csv.py mps.json mps.csv`.
```

Comment:
Perhaps we could make scrapy output to stdout instead than we could formulate this as something like:

```bash
scrapy runspider spider.py -o - | python3 json2csv.py > mps.csv
```

Reply:
The optimum would probably be one (click?) command that handles all of this and outputs a CSV. Then we could install this a requirement in e2t-data.